src/common/state/db_client_table.cc

```diff
@@ -27,3 +27,13 @@ void db_client_table_subscribe(
                       (table_done_callback) done_callback,
                       redis_db_client_table_subscribe, user_context);
 }
+
+void plasma_manager_send_heartbeat(DBHandle *db_handle) {
+  RetryInfo heartbeat_retry = {.num_retries = 0,
+                               .timeout = HEARTBEAT_TIMEOUT_MILLISECONDS,
+                               .fail_callback = NULL};
```

Comment:
Would you mind replacing this with

```
RetryInfo heartbeat_retry;
heartbeat_retry.num_retries = 0;
heartbeat_retry.timeout = HEARTBEAT_TIMEOUT_MILLISECONDS;
heartbeat_retry.fail_callback = NULL;
```

Note that we use the default retry info for the local scheduler heartbeats. I agree retrying doesn't make sense (anyway, I'm not suggesting that we change it here).